Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/ViewportEditorModeTracker.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/RTTI/RTTI.h>
+#include <AzCore/std/containers/array.h>
+#include <AzCore/std/containers/unordered_map.h>
+#include <AzToolsFramework/API/ViewportEditorModeTrackerNotificationBus.h>
+#include <AzToolsFramework/API/ViewportEditorModeTrackerInterface.h>
+
+namespace AzToolsFramework
+{
+    //! The encapsulation of the editor modes for a given viewport.
+    class ViewportEditorModes
+        : public ViewportEditorModesInterface
+    {
+    public:
+
```

Comment:
uber nit: just for consistency could eliminate this blank line (and have a break line break before `private` lower down (`ViewportEditorModeTracker` does this below)

Reply:
fixed 👍